modules/ref_ipi-installation-overview.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/ipi/ipi-install-prerequisites.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="installation-overview_{context}"]
+= Installation overview
+
+The installation program supports interactive mode, but Red{nbsp}Hat recommends preparing an `install-config.yaml` file in advance, containing the provisioning details for all of the bare-metal hosts, and the relevant cluster details.
+
+The installation program loads the `install-config.yaml` file, while the administrator generates the manifests and verifies all prerequisites.
```

Comment:
Per ISG: Use _while_ only to refer to a period of time...
```suggestion
The installation program loads the `install-config.yaml` file and the administrator generates the manifests and verifies all prerequisites.
```

Reply:
Updated as suggested.